logging/FindLog4cplus.cmake

```diff
@@ -0,0 +1,84 @@
+
+#--------------------------------------------------------------------------------
+# Copyright (c) 2012-2013, Lars Baehren <lbaehren@gmail.com>
```

Comment:
Please make sure we are allowed to use this file

Reply:
I'm replacing this file by cmake code written entirely by me :-)

Reply:
done